### PR TITLE
Several improvements for the admin module

### PR DESCRIPTION
**Pull request by GilianJuice**
* It's now possible to move entity properties to a different tab.
* It's now possible to move entire tabs (with all it's properties) to a different position.
* Fixed problem that a lot of empty properties were saved in options in wiser_entityproperty.
* Fixed error that sometimes occurred when loading a field. Some Wiser instances apparently don't have the column "link_type" of "wiser_entityproperty" set to NOT NULL.
* Fixed problem that moving properties to a different position did not work correctly and cause duplicate order numbers. Also refactored the code.
* Added check to warn the user if they have unsaved changes when they're about to leave the page. This only works for the first tab at the moment.
* Added several missing options for combobox fields.
* Fixed javascript error that occurred when loading a combobox field for the first time.

Asana ticket: https://app.asana.com/0/1201027711166952/1201693099409559

### CODE REVIEW

File: FrontEnd/Modules/Admin/Scripts/EntityTab.js [hunk 1]
```diff
@@ -3136,39 +3151,33 @@ export class EntityTab {
...
-    afterSave(entityProperties) {
-        const selectCorrectListViewItem = () => {
-            // select correct entity display name
-            this.listOfTabProperties.one("dataBound",
-                () => {
-                    this.selectPropertyInListView(entityProperties.displayName);
-                });
-        };
+    async afterSave(property) {
+        console.log("property", property);
```
Comment (MarkVanDijkHG) [inline FrontEnd/Modules/Admin/Scripts/EntityTab.js:3155]: Please remove the console log.

File: Api/Modules/EntityProperties/Services/EntityPropertiesService.cs [hunk 2]
```diff
@@ -692,6 +717,172 @@ public async Task<ServiceResult<bool>> FixOrderingAsync(ClaimsIdentity identity,
...
+                var orderingPropertyName = $"ordering{index}";
+                var idPropertyName = $"id{index}";
+                clientDatabaseConnection.AddParameter(orderingPropertyName, newOrdering);
+                clientDatabaseConnection.AddParameter(idPropertyName, propertyId);
+                queries.Add($"UPDATE {WiserTableNames.WiserEntityProperty} SET ordering = ?{orderingPropertyName} WHERE id = ?{idPropertyName};");
+            }
+
+            await clientDatabaseConnection.ExecuteAsync(String.Join(Environment.NewLine, queries));
+
+
```
Comment (MarkVanDijkHG) [inline Api/Modules/EntityProperties/Services/EntityPropertiesService.cs:882]: Please remove the second new line.

**Review by MarkVanDijkHG — CHANGES_REQUESTED** [review]
2 small changes.

File: FrontEnd/Modules/Admin/Scripts/EntityTab.js (hunk 1 above)
Comment (GilianJuice) [reply FrontEnd/Modules/Admin/Scripts/EntityTab.js]: I removed the console.log

File: Api/Modules/EntityProperties/Services/EntityPropertiesService.cs (hunk 2 above)
Comment (GilianJuice) [reply Api/Modules/EntityProperties/Services/EntityPropertiesService.cs]: I removed the extra new line.

**Comment by GilianJuice** [comment]
I merged develop into this branch and fixed merge conflicts.

**Review by MarkVanDijkHG — APPROVED** [review]

**Pull request merged**